src/App.tsx

```diff
@@ -7,27 +7,76 @@ import Oppfolging from './components/oppfolging';
 import PersonaliaBoks from './components/personalia-boks';
 import { Registrering } from './components/registreringsInfo';
 import { Ytelser } from './components/ytelserinfo';
+import { Chips } from '@navikt/ds-react';
+import { useState } from 'react';
 
 export interface AppProps {
     fnr: string;
     enhet?: string;
 }
 
-const App = (props: AppProps) => (
-    <main className="app veilarbdetaljerfs">
-        <StoreProvider fnr={props.fnr}>
-            <PilotAlert />
-            <Nokkelinfo />
-            <div className="main_grid">
-                <CvInnhold />
-                <Jobbonsker />
-                <Oppfolging />
-                <PersonaliaBoks />
-                <Registrering />
-                <Ytelser />
-            </div>
-        </StoreProvider>
-    </main>
-);
+const App = (props: AppProps) => {
+    const options = ['CV', 'Jobbønsker', 'Oppfølging', 'Personalia', 'Registrering', 'Ytelser'];
+
+    const [selectedComponents, setSelectedComponents] = useState<string[]>(options);
+
+    const toggleComponent = (componentName: string) => {
+        setSelectedComponents((prevSelected) =>
+            prevSelected.includes(componentName)
+                ? prevSelected.filter((name) => name !== componentName)
+                : [...prevSelected, componentName]
+        );
+    };
+
+    const renderComponent = (componentName: string) => {
+        switch (componentName) {
+            case 'CV':
+                return <CvInnhold />;
+            case 'Jobbønsker':
+                return <Jobbonsker />;
+            case 'Oppfølging':
+                return <Oppfolging />;
+            case 'Personalia':
+                return <PersonaliaBoks />;
+            case 'Registrering':
+                return <Registrering />;
+            case 'Ytelser':
+                return <Ytelser />;
+            default:
+                return null;
+        }
+    };
```

Comment:
Forslag til navneendring som kanskje gir litt mer mening:

```suggestion
    const mapNavnTilKomponent = (navn: string) => {
        switch (navn) {
            case 'CV':
                return <CvInnhold />;
            case 'Jobbønsker':
                return <Jobbonsker />;
            case 'Oppfølging':
                return <Oppfolging />;
            case 'Personalia':
                return <PersonaliaBoks />;
            case 'Registrering':
                return <Registrering />;
            case 'Ytelser':
                return <Ytelser />;
            default:
                return null;
        }
    };
```